tests/tests/test-precompiles.ts

```diff
@@ -34,7 +34,8 @@ describeWithMoonbeam("Moonbeam (Precompiles)", `simple-specs.json`, (context) =>
 
 
 
-    it("ripemd1620 is valid inside a contract", async function () {
+	// TODO: Restore this test once manual sealing is fixed
```

Comment:
```suggestion
	// TODO: Restore this test once manual sealing is fixed https://purestake.atlassian.net/browse/MOON-81
```

Is it better to add a delay before authoring the block, than to skip the test entirely?

Reply:
I see you merged my suggestion, but I am curious what you think about adding a manual delay during the test. It definitely isn't a long term solution, but is better than skipping the test entirely, right?